src/components/auth/forgot_password/ResetPasswordForm.js

```diff
@@ -0,0 +1,166 @@
+// @flow
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { push } from 'connected-react-router';
+import humps from 'humps';
+import qs from 'qs';
+import { Link } from 'react-router-dom';
+import Form from 'react-validation/build/form';
+import Input from 'react-validation/build/input';
+import Button from 'react-validation/build/button';
+import UserService from '../../../services/UserService';
+import ValidationErrors from '../../utility/form/ValidationErrors';
+import type { Errors } from '../../../models/Errors';
+import '../Forms.scss';
```

Comment:
Actually, you don't need to import this css all the time. It's enough to load it once and properly better.
Consider loading it in `index.scss`.